pages/common/pup.md

```diff
@@ -0,0 +1,27 @@
+# pup
+
+> Command line HTML parsing tool.
+
+- Transform a raw HTML file into a cleaned, indented, and colored format:
+
+`cat {{index.html}} | pup --color`
+
+- Filter HTML by element tag name:
+
+`cat {{index.html}} | pup '{{div}}'`
```

Comment:
We can keep this as {{tag}} to match with "element tag name". Since we are giving a lot of concrete examples later anyway.